### PR TITLE
dev foundations article

### PR DESCRIPTION
**Pull request by billkable**
A new [Outcomes Series](https://tanzu.vmware.com/developer/outcomes/) for Modern Application Development workflow, and the first *Foundations* article.

### CODE REVIEW

File: content/outcomes/application-development-rhythm/_index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Who is this learning path for?
+
+This learning path is for software engineers and other technical
+development team members involved in the construction of modern
+applications.
+
+Managers, designers, and other non-technical stakeholders may also find
+this learning path useful for understanding how modern application
+developer do their daily work.
```
Comment (joemoore) [inline content/outcomes/application-development-rhythm/_index.md:35]:
```suggestion
developers do their daily work.
```

File: content/outcomes/application-development-rhythm/foundations.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+This article outlines the context,
+as well as the core building blocks necessary to sustain a product or
+project using modern application development practices.
+
+## What you will learn
+
+In this article you will be able to:
+
+-   Explain the context of modern application development.
+-   The types of building blocks that comprise modern application
```
Comment (joemoore) [inline content/outcomes/application-development-rhythm/foundations.md:23]: Is this "Explain the types" or "Describe the types" or "List the types"?

File: content/outcomes/application-development-rhythm/foundations.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+live.
+The modern application development practitioner may use multiple
+practices during their daily workflow to solve problems efficiently and
+safely.
+
+Some of the practices you will learn about in this series are:
+
+-   [Pair Programming](http://localhost:1313/developer/outcomes/application-development/pair-programming/)
+
+-   [Test Driven Development](http://localhost:1313/developer/outcomes/application-development/test-driven-development/)
```
Comment (joemoore) [inline content/outcomes/application-development-rhythm/foundations.md:143]: locahost links

File: content/outcomes/application-development-rhythm/foundations.md (hunk 2 above)
Comment (billkable) [reply content/outcomes/application-development-rhythm/foundations.md:23]: @joemoore Describe the types.  Thanks for the catch.

File: content/outcomes/application-development-rhythm/foundations.md (hunk 3 above)
Comment (billkable) [reply content/outcomes/application-development-rhythm/foundations.md]: @joemoore Fixed per [c058a65](https://github.com/vmware-tanzu/tanzu-dev-portal/pull/1122/commits/c058a6583e50073bd4e34bfe381b002eea84cb7c).

**Comment by billkable** [comment]
@BrianMMcClain Can you review and merge?  I am done with all the suggested review edits.

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**